### PR TITLE
Adding walkthrough

### PR DESCRIPTION
**Pull request by RossKen**
Found 'rintrojs' package that allows you to do walkthrough tutorials in shiny apps (amazing, i know) so trying to set this up for PQ Tool

### CODE REVIEW

**Comment by EdwardAndress** [comment]
During the tour, unless we put some example text in the search field, the dat-inputs are hidden.

screen shot 2017-08-09 at 11 00 00

**Comment by EdwardAndress** [comment]
So is the table

**Comment by EdwardAndress** [comment]
Ditt for everything else – am I doing something wrong?

**Comment by RossKen** [comment]
@EdwardAndress no, you are right. The tutorial only really works if you follow the instructions (i.e. enter search terms etc). Otherwise it breaks

Am looking into a way of forcing the user to do these things before being allowed to hit 'next'

**Comment by RossKen** [comment]
An alternative to this is to to take the interactivity out of the walkthrough so you can just hit next and go through all the features - but I feel this is less effective in engaging the user

**Comment by EdwardAndress** [comment]
Is it possible to add some text in, automatically, as the user clicks through?

**Comment by RossKen** [comment]
I think so, but would have to play about with it

File: server.R [hunk 1]
```diff
@@ -182,39 +183,65 @@ function(input, output, session) {
...
+      introjs(session,
+              events = list(
+                "onchange" = I("console.log(this._currentStep)
+                              if (this._currentStep==6) { 
+                               $('.btn-info')[0].addEventListener('mouseup', function(){
+                               setTimeout(function(){
+                               $('.introjs-nextbutton').click()
+                               }, 1000)
+                               })
+                               } else if (this._currentStep==10) { debugger
```
Comment (EdwardAndress) [inline server.R]: is `debugger` supposed to be here?

Comment (RossKen) [inline server.R]: It has no impact on the functionality of the tool unless you have developer tools open in your browser, but you're right. Should be removed

**Comment by RossKen** [comment]
screen shot 2017-08-09 at 11 25 50

**Comment by EdwardAndress** [comment]
Thanks Ross, I will continue looking into this

**Comment by EdwardAndress** [comment]
Looks like I broke something!

**Comment by RossKen** [comment]
This PR is superseded by Fix Walkthrough

**Pull request closed without merging**